plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Doxygen.java

```diff
@@ -28,7 +30,7 @@ public Doxygen() {
 
     @Override
     public IssueParser createParser() {
-        return new Gcc4CompilerParser();
+        return new Gcc4CompilerParser(DOXYGEN_WARNING_PATTERN);
```

Comment:
Replace with new `DoxygenParser`. Additionally, the analysis-model version needs to be changed in the `pom.xml`.

Reply:
:+1: